### ISSUE TITLE
Split apart e2e tests for the different sources

### ISSUE THREAD

**Issue by tlater-famedly** [famedly/ldap-sync#60]
I added a test that combines LDAP and UKT syncing into one as per your suggestion. Also, as per your suggestion in another comment, let's postpone the division of tests for another PR.

_Originally posted by @jannden in https://github.com/famedly/ldap-sync/pull/49#discussion_r1759251203_

This would be nice to do before we start adding more sources, as the e2e test file will just keep growing if we don't get to this.
